src/milia/api/widgets.cljc

```diff
@@ -28,7 +27,10 @@
     `:widget_type` is a string, determined by the client e.g. chart
     `:view_type` is a string, determined by the client e.g. bar-chart
     `:column` is the  data column to be stored based on the form field.
-    `:group_by` the data column for the data to be grouped by. Optional"
+    `:order` this is the position of the widget in relation to others
```

Comment:
Cool, yeah just checked the tests to make sure, lets also add "Optional" at the end of this arg description